app/assets/javascripts/widgets/wysiwyg.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready(function(){
+  var anchorLinks = $('h3.anchorLinks a, a[href="#top"]');
+  var navHeight = $("#cu_nav").outerHeight();
+
+  /* offset omninav when interal anchor links scroll to target */
+  anchorLinks.on('click', function(e){
+    var target = $('[name=' + this.hash.slice(1) + ']');
```

Comment:
I just named the variable target because it is the scroll target. `this` is referring to the anchor element that was clicked. and `this.hash` returns the anchor part of a url.